Delegation documentation (#448)
Concordium/concordium.github.io: source/shared/net/desktop-wallet/add-delegation.rst

#. A message says **Waiting for device. Please connect your Ledger**. Connect the Ledger device to the computer and enter your PIN on the Ledger device.

#. Press the right button to navigate to the **Concordium** app, and then press both buttons to open the app. The Ledger says **Concordium is ready**. Wait for the message **Ledger Nano S is ready** in the Desktop Wallet and select **Submit**.

#. On the Ledger device, a message says **Review transaction**. Verify that the sender account is correct and navigate to the right. Verify that the Ledger shows the correct amount to delegate and navigate to the right. Verify that the restake preference is correct and navigate to the right. Verify that the delegation target is correct and navigate to the right.

#. The Ledger device says **Sign transaction**. Press both buttons to sign the transaction. The Ledger device says **Concordium is ready**.

#. In the Desktop Wallet, you can see that the transaction has been submitted to the chain. Select **Finish**.

Delegate to a baker pool (Multi-signature account)
==================================================

> #. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Update baker stake**.

Reviewer: ```suggestion
#. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Delegate to a pool**.
```

DW says that, but maybe it should actually say "Register as a delegator".

> #. Select the account whose baker stake you want to change.

Reviewer: ```suggestion
#. Select the account on which you want to delegate funds.
```

> #. Enter the new amount that you want to stake and select **Continue**.

Reviewer: ```suggestion
#. Enter the new amount that you want to stake and select **Continue**. Enter the amount that you want to delegate and choose whether rewards should be redelegated or not. Click **Continue**.
```

#. In the Desktop Wallet, Verify that the **Transaction details** are as you intended, select **I am sure that the proposed changes are correct**, and then select **Generate and Sign**.

#. On the Ledger, there's a message saying **Review transaction**. Verify that the sender account is correct and navigate to the right. Verify that the Ledger shows the correct amount to delegate and navigate to the right. Verify that the restake preference is correct and navigate to the right. Verify that the delegation target is correct and navigate to the right.

#. The Ledger says **Sign transaction**. Press both buttons to sign the transaction. The Ledger says **Concordium is ready**.

.. Note::
   If you want to decline the transaction, press the right button on the Ledger. The hardware wallet now says **Decline to sign transaction**. Press both buttons to decline. In the Desktop Wallet, there's a message saying **The action was declined on the Ledger device. Please try again.**

> In the Desktop Wallet, you can now see **Transaction details**, **Signatures**, and **Security & Submission Details**, which include the status of the transaction, the identicon, and the transaction hash. If you have all the required signatures, you can :ref:`submit the transaction to the chain <submit-stake-change>`, otherwise, you'll have to export the proposal and receive signatures from the co-signers.

Reviewer: I don't what the ref should be to refer to the submit part of the current document, but currently it links to the update stake page. (submit-stake-change).
Same for the other delegation transaction.

You can delegate stake from an account to a baker pool or to passive delegation.

When you delegate some stake to a baker pool, it can influence the chances of the baker owner of the baker pool being selected to bake a block and receive baker rewards and thus delegation rewards to you.

.. Note::

   All transfers and transactions cost a fee, including staking and unstaking transactions. The fee is based on the set NRG for that transaction and the current exchange rate.
   The cost of transaction fees is stable in Euros, and therefore the price in CCD varies depending on the CCD to EUR exchange rate. The fee will always be deducted from the **Balance** of the account, so it is important to have some available CCDs to cover fees. A locked-for-staking balance cannot be used to pay for these transactions.
   You can see the fee in the transaction log.

.. Warning::
   Make sure you have enough funds in your disposable balance to cover transaction fees.

Delegate to a baker pool (Single-signature account)
===================================================

#. Go to **Accounts** and select the account on which you want to delegate funds.

> #. Click **More options** then select **Register to a delegator**.

Reviewer 2: ```suggestion
#. Click **More options** then select **Register as a delegator**.
```
